framework/src/modules/chain/submodules/blocks.js

```diff
@@ -78,7 +78,8 @@ class Blocks {
 				scope.schema,
 				scope.components.storage,
 				scope.sequence,
-				scope.genesisBlock
+				scope.genesisBlock,
+				scope.channel
```

Comment:
Why do we need this?

Reply:
Because the file `lisk/framework/src/modules/chain/submodules/blocks/process.js` needs to invoke a network action - This may be refactored later but right now it's the simplest thing to do.